modules/ROOT/pages/supported-authentication-methods.adoc

```diff
@@ -30,9 +30,19 @@ curl GET 'http://localhost/myResource' \
   "password": "myPassword"
 }'
 --
+
+* *Pass-through authentication*
```

Comment:
@mauro-dentoni this is for pass-through auth docs on the supported authentication methods page

Reply:
Looks fine